GherkinSyncTool/GherkinSyncTool/Configuration/Config.cs

```diff
@@ -13,6 +13,7 @@ public class Config
         public string TestRailBaseUrl { get; set; } 
         public string TestRailUserName { get; set; }
         public string TestRailPassword { get; set; }
+        public int? MaximumRequestsPerMinute { get; set; }
```

Comment:
`TestRailMaximumRequestsPerMinute` or we can discuss a separate object for TestRail configs.